misc/formatter/modules/formatter-core/src/main/java/org/ballerinalang/formatter/core/ForceFormattingOptions.java

```diff
@@ -0,0 +1,53 @@
+/*
+ * Copyright (c) 2022, WSO2 Inc. (http://wso2.com) All Rights Reserved.
```

Comment:
Let's modify all the new license headers as follows.

```suggestion
 * Copyright (c) 2022, WSO2 LLC. (http://wso2.com) All Rights Reserved.
```